source/_components/hdmi_cec.markdown

```diff
@@ -30,13 +30,13 @@ The computer running Home Assistant must support CEC, and of course be connected
 Create a symlink to the `cec` installation.
  
 ```bash
-$ ln -s /usr/local/lib/python3.4/dist-packages/cec /path/to/your/venv/lib/python3.4/site-packages
+$ ln -s /usr/local/lib/python3.4/site-packages/cec /path/to/your/venv/lib/python3.4/site-packages
 ```
 
 For the default virtual environment of a [Raspberry Pi All-In-One installation](/getting-started/installation-raspberry-pi-all-in-one/) the command would be as follows.
 
 ```bash
-$ ln -s /usr/local/lib/python3.4/dist-packages/cec /srv/hass/hass_venv/lib/python3.4/site-packages
+$ ln -s /usr/local/lib/python3.4/site-packages/cec /srv/homeassistant/homeassistant_venv/lib/python3.4/site-packages
```

Comment:
looks like there is a commit in the "next" branch that clears up things some

https://github.com/matt-cahill/home-assistant.github.io/commit/c194ab6b7ae2800db499b4099554b6604d0fdeda